### PR TITLE
fix(bat): don't use IFD

### PR DESCRIPTION
**Pull request by lovesegfault**
Blocked on: https://github.com/nix-community/home-manager/pull/4539

### CODE REVIEW

**Review by danth — APPROVED** [review]
Looks good, ready to merge once the pull request is merged upstream.

**Comment by lovesegfault** [comment]
@danth Upstream PR is merged, this is ready to go :)

**Pull request merged**
